controllers/user.js

```diff
@@ -1813,6 +1812,46 @@ async function createRemovalHashWaitlist(req, res) {
   });
 }
 
+async function createRemovalHashAdminList(req, res) {
```

Comment:
We can probabaly just get rid of this whole create-a-file step. We could add a single env var:
```
REMOVAL_ADMINS=26bb92cc8f4c7be7ea2adb4611fd713358eaf128,12eeb998fe0c7600b3a3aacf518aa760b1dbd775,df79383a705df15058f290aad1bb0f4c382a81ec
```